pkg/annotations/ingress_annotations.go

```diff
@@ -40,13 +41,17 @@ const (
 	// that this is an ingress resource meant for the application gateway ingress controller.
 	IngressClassKey = "kubernetes.io/ingress.class"
 
+	// IstioGatewayKey defines the key of the annotation which needs to be set in order to specify
+	// that this is sa gateway meant for the application gateway ingress controller.
+	IstioGatewayKey = "appgw.ingress.istio.io/v1alpha3"
+
 	// ApplicationGatewayIngressClass defines the value of the `IngressClassKey` annotation that will tell the ingress controller
 	// whether it should act on this ingress resource or not.
 	ApplicationGatewayIngressClass = "azure/application-gateway"
 
 	// ApplicationGatewayIstioIngressClass defines the value of the `IngressClassKey` annotation that will tell the ingress controller whether
-	// this ingress resource is from Istio or not.
-	ApplicationGatewayIstioIngressClass = "istio-ingress"
+	// this gateway should be handled by AGIC or not
+	ApplicationGatewayIstioIngressClass = "controls-app-gw"
```

Comment:
can we use the same ingress class for Istio ?

Reply:
I'm not sure - the Gateway I'm playing around with right now doesn't have an ingress class annotation, so I'm not sure if it's necessary, but I can add that in later if it is.